sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityTests.java

```diff
@@ -78,11 +80,39 @@ public void createUserWithResponseNullContext() {
     }
 
     @Test
-    public void createUserAndToken() {
+    public void createUserAndTokenWithChatScope() {
+        // Arrange
+        CommunicationIdentityClientBuilder builder = createClientBuilder(httpClient);
+        client = setupClient(builder, "createUserAndTokenSync");
+        List<CommunicationTokenScope> scopes = Arrays.asList(CHAT);
+
+        // Action & Assert
+        CommunicationUserIdentifierAndToken result = client.createUserAndToken(scopes);
+        assertNotNull(result.getUser().getId());
+        assertNotNull(result.getUserToken());
```

Comment:
```suggestion
        verifyTokenNotEmpty(result.getUserToken());
```
And add the `assertNotNull` check to the `verifyTokenNotEmpty` method. In this way, we can verify not only the access token is not null but also the variables of the AccessToken class, the same comment for the async test